roles/satosa/templates/saml2_backend.yaml.j2

```diff
@@ -1,27 +1,45 @@
+---
 module: satosa.backends.saml2.SAMLBackend
 name: SamlSP
 config:
   sp_config:
     key_file: certs/backend.key
     cert_file: certs/backend.crt
-    organization: {display_name: {{ environment_string }} Proxy SP, name: {{ environment_string }} Proxy SP, url: 'https://{{ hostnames.meta }}/'}
+    organization:
+      display_name: {{ environment_string }} Proxy SP
+      name: {{ environment_string }} Proxy SP
+      url: 'https://{{ hostnames.meta }}/'
     contact_person:
-    - {contact_type: technical, email_address: technical@{{ hostnames.proxy }}, given_name: Technical}
-    - {contact_type: support, email_address: support@{{ hostnames.proxy }}, given_name: Support}
+      - contact_type: technical
+        email_address: technical@{{ hostnames.proxy }}
+        given_name: Technical
+      - contact_type: support
+        email_address: support@{{ hostnames.proxy }}
+        given_name: Support
```

Comment:
Contacts should also be configurable.  Maybe just specify them in the group_vars like
```yaml
metadata_contacts:
  - { type: "technical", name: "Pietje Puk", email: "pietje@example.org" }
  - { type: "support", name: "Jantje Jansen", email: "jantje@example.org" }
```